DonjinKrawler-Client/src/main/java/donjinkrawler/flyweight/EnemyType.java

```diff
@@ -0,0 +1,59 @@
+package donjinkrawler.flyweight;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class EnemyType {
```

Comment:
Any particular reason why we need a duplicate EnemyType class? Maybe the naming should be better, since this doesn't represent the enemy type at all

Reply:
I'll rename it to EnemyFlyweight.